lib/runnable.js

```diff
@@ -40,6 +40,7 @@ function Runnable(title, fn) {
   this._retries = -1;
   this._currentRetry = 0;
   this.pending = false;
+  this.skipped = false;
```

Comment:
You know, if we have `Test#state`, maybe we should have `Runnable#state` instead, and do away with these booleans...?